update snapshot options and add env vars (#499)
meilisearch/documentation: guides/advanced_guides/configuration.md

If this command is not called, no snapshot will be imported.

### Ignore missing snapshot

**Environment variable**: `MEILI_IGNORE_MISSING_SNAPSHOT`
**CLI option**: `--ignore-missing-snapshot`

> The engine will ignore a missing snapshot and not return an error in such case.

Reviewer: ```suggestion
The engine ignores missing snapshots and does not throw an error in this case.
```
What do you mean by that? 
If `--load-from-snapshot` is enabled but their are no snapshots in the `--snapshot-path` then their will be no errors thrown and no snapshots will be loaded? Is this correct?

### Ignore snapshot if db exists

**Environment variable**: `MEILI_IGNORE_SNAPSHOT_IF_DB_EXISTS`
**CLI option**: `--ignore-snapshot-if-db-exists`

> The engine will skip snapshot importation if a database already exists and not return an error in such case.

Reviewer: ```suggestion
The engine skips snapshot importation if a database already exists. No error is thrown in this case.
```
